src/cpp/r/RUtil.cpp

```diff
@@ -72,6 +80,79 @@ bool versionTest(const std::string& comparator, const std::string& version)
 
 } // anonymous namespace
 
+void setenv(const std::string& key, const std::string& value)
```

Comment:
Is this something where we should potentially be replacing calls to core::system::setenv with calls to rutil::setenv in a loooooooot of places in the code?

Reply:
Potentially yes. 😞 

It's not completely obvious where we should be making these changes, since some calls to `core::system::setenv()` in the R session codebase run before the R session has been initialized, and I think those usages are safe (and in fact we wouldn't be able to use `r::util::setenv()` since it wouldn't be safe to call into R before it's been initialized)